docs/release_guide.md

```diff
@@ -52,10 +52,10 @@ be made, they should be numbered as 2041.01.1, 2041.01.2, etc.
 
         git tag -as 2021.10
 
-9. Run `git verify-tag 2021.10` to make sure the signature is valid and the tag
+9. Run `git verify-tag 2024.05` to make sure the signature is valid and the tag
    was actually signed.
 
-10. Run `git push origin refs/heads/2021.10:refs/heads/2021.100` assuming
+10. Run `git push origin refs/heads/2021.05:refs/heads/2024.05` assuming
```

Comment:
Stop the presses! Do not approve! Broken PR! Oh the humanity! “2021.05” -> “2024.05”

Reply:
Now that you mention it, I wasn't sure if `2021.100` was a typo in the original, or if there is meant to be an additional `0`.